driver-core/pom.xml

```diff
@@ -54,11 +54,6 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
-        <dependency>
```

Comment:
When a project uses some dependency directly, it is usually recommended that it declares that dependency explicitly instead of relying on transitive dependencies. I would prefer that we keep the dependency declared here.

Reply:
I considered that, but I was sort of put off by how jnr-posix upgraded minor version dependency of jnr-ffi in a patch release.  Since we now have a test to verify the functionality of jnr-posix retrieving pids, we could add the direct dependency back and feel relatively safe, so i'll do that.